src/main/java/io/github/thebusybiscuit/sensibletoolbox/listeners/ElevatorListener.java

```diff
@@ -65,4 +67,10 @@ public void onPlayerSneak(PlayerToggleSneakEvent event) {
         }
     }
 
+    private boolean isOnGround(@Nonnull Player player) {
+        Location location = player.getLocation();
+        Block currentBlock = location.getBlock();
+    	return !currentBlock.getRelative(BlockFace.DOWN).isEmpty() && location.getY() == location.getBlockY();
```

Comment:
I am feeling comfortable with these changes overall.
However one thing I would definitely like to have changed before merging this is the floating point comparison at the end.
`.getY()` will return a floating point value (double), so this comparison will be subject to floating point value errors.
I think the best way to circumvent this would be to check if the distance between these two numbers is below a very small number.

Reply:
How small would be comfortable? In my testing this worked as expected but I understand your concern. Would something like 0.05 suffice or would you prefer something lower?

Reply:
0.05 would be fine

Reply:
I think something like 0.001 or such might be required, but feel free to test out 2 or 3 values. It should approximate zero, that's all.

Reply:
> I think something like 0.001 or such might be required, but feel free to test out 2 or 3 values. It should approximate zero, that's all.

That's not that required actually. Even 0.05 means less than a pixel of a block. If you are that low you practically are on ground.

Reply:
Oh okay then